README.md

```diff
@@ -87,17 +87,40 @@ $address->setCountryCode('US');
 $address->setPostalCode('10000');
 
 $xav = new \Ups\AddressValidation($accessKey, $userId, $password);
+
+$xav->validateReturnAVObject();
```

Comment:
Would you prefer the comments about it being optional in the code sample, or in the docs that tries to explain what that function does below the code sample?